pkg/capabilities/pb/capabilities_helpers.go

```diff
@@ -1,9 +1,101 @@
 package pb
 
+import (
+	"errors"
+
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
 const (
 	CapabilityTypeUnknown   = CapabilityType_CAPABILITY_TYPE_UNKNOWN
 	CapabilityTypeTrigger   = CapabilityType_CAPABILITY_TYPE_TRIGGER
 	CapabilityTypeAction    = CapabilityType_CAPABILITY_TYPE_ACTION
 	CapabilityTypeConsensus = CapabilityType_CAPABILITY_TYPE_CONSENSUS
 	CapabilityTypeTarget    = CapabilityType_CAPABILITY_TYPE_TARGET
 )
+
+func MarshalCapabilityRequest(req capabilities.CapabilityRequest) ([]byte, error) {
+	return proto.Marshal(CapabilityRequestToProto(req))
+}
+
+func CapabilityRequestToProto(req capabilities.CapabilityRequest) *CapabilityRequest {
+	inputs := &values.Map{Underlying: map[string]values.Value{}}
+	if req.Inputs != nil {
+		inputs = req.Inputs
+	}
+	config := &values.Map{Underlying: map[string]values.Value{}}
+	if req.Config != nil {
+		config = req.Config
+	}
+	return &CapabilityRequest{
+		Metadata: &RequestMetadata{
+			WorkflowId:          req.Metadata.WorkflowID,
+			WorkflowExecutionId: req.Metadata.WorkflowExecutionID,
+		},
+		Inputs: values.Proto(inputs),
+		Config: values.Proto(config),
+	}
+}
+
+func UnmarshalCapabilityRequest(raw []byte) (capabilities.CapabilityRequest, error) {
+	var cr CapabilityRequest
+	if err := proto.Unmarshal(raw, &cr); err != nil {
+		return capabilities.CapabilityRequest{}, err
+	}
+	return CapabilityRequestFromProto(&cr), nil
+}
+
+func CapabilityRequestFromProto(pr *CapabilityRequest) capabilities.CapabilityRequest {
+	md := pr.Metadata
+	config := values.FromProto(pr.Config)
+	inputs := values.FromProto(pr.Inputs)
+
+	return capabilities.CapabilityRequest{
+		Metadata: capabilities.RequestMetadata{
+			WorkflowID:          md.WorkflowId,
+			WorkflowExecutionID: md.WorkflowExecutionId,
+		},
```

Comment:
Could we direct assign here rather than make a new struct?

Reply:
we can't - these are different structs even though they are named the same